test/common/ssl/test_data/certs.sh

```diff
@@ -13,6 +17,21 @@ EOF
 
 openssl x509 -req -days 3650 -in ca_cert.csr -sha256 -signkey ca_key.pem -out ca_cert.pem
 
+# Generate fake_ca_key.pem and fake_ca_cert.pem (intentionally pretending CA above).
```

Comment:
Could you remove fake_ca_key.pem and also xxx_key.pem from other comments? We do not generate the key files in this file.